package.json

```diff
@@ -21,20 +23,24 @@
     "@typescript-eslint/eslint-plugin": "^3.1.0",
     "@typescript-eslint/parser": "^3.1.0",
     "coveralls": "^3.1.0",
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^3.0.2",
     "eslint": "^7.2.0",
     "eslint-config-prettier": "^6.11.0",
     "husky": "^4.2.5",
     "jest": "^26.0.1",
     "lint-staged": "^10.2.9",
     "mysql2": "^2.1.0",
     "prettier": "^2.0.5",
-    "rimraf": "^3.0.2",
     "ts-jest": "^26.1.0",
     "tslint": "^6.1.2",
-    "typescript": "^3.9.5"
+    "typescript": "^3.9.5",
+    "casbin": "^5.0.3"
```

Comment:
Why?

Reply:
This can reduce the size of `node_modules`. If place it in the dependecies, user will have 2 node-casbin at the same time.

Reply:
@Gabriel-403 @Zxilly why don't them follow the alphabet order?